src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionGeneralSettings.tag

```diff
@@ -0,0 +1,227 @@
+                <th class="text-center">Can see answer</th>
+                <th class="text-center">Can see giver's name</th>
+                <th class="text-center">Can see recipient's name</th>
+            </tr>
+            <tr>
+                <td class="text-left">
+                    <div data-toggle="tooltip" data-placement="top" title="<%= Const.Tooltips.VISIBILITY_OPTIONS_RECIPIENT %>">
+                        Recipient(s)
+                    </div>
+                </td>
+                <td>
+                    <input class="visibilityCheckbox answerCheckbox${fqForm.question.questionNumber} centered"
+                        name="receiverLeaderCheckbox" type="checkbox"
+                        value="<%= FeedbackParticipantType.RECEIVER %>" <c:if test="!fqForm.isEditable">disabled="disabled"</c:if>
+                        <c:if test="${fqForm.generalSettings.responseVisibleFor['RECEIVER']}"> checked="checked"</c:if> >
+                </td>
+                <td>
+                    <input class="visibilityCheckbox giverCheckbox${fqForm.question.questionNumber}"
+                        type="checkbox" value="<%= FeedbackParticipantType.RECEIVER %>" <c:if test="!fqForm.isEditable">disabled="disabled"</c:if>
+                        <c:if test="${fqForm.generalSettings.giverNameVisibleFor['RECEIVER']}"> checked="checked"</c:if> >
+                </td>
+                <td>
+                    <input class="visibilityCheckbox recipientCheckbox${fqForm.question.questionNumber}"
+                        name="receiverFollowerCheckbox" type="checkbox"
+                        value="<%= FeedbackParticipantType.RECEIVER %>" disabled="disabled"
+                        <c:if test="${fqForm.generalSettings.recipientNameVisibleFor['RECEIVER']}"> checked="checked"</c:if> >
+                </td>
+            </tr>
+            <tr>
+                <td class="text-left">
+                    <div data-toggle="tooltip" data-placement="top" title="<%= Const.Tooltips.VISIBILITY_OPTIONS_GIVER_TEAM_MEMBERS %>">
+                        Giver's Team Members
+                    </div>
+                </td>
+                <td>
+                    <input class="visibilityCheckbox answerCheckbox${fqForm.question.questionNumber}"
+                        type="checkbox" value="<%= FeedbackParticipantType.OWN_TEAM_MEMBERS %>" <c:if test="!fqForm.isEditable">disabled="disabled"</c:if>
+                        <c:if test="${fqForm.generalSettings.responseVisibleFor['OWN_TEAM_MEMBERS']}"> checked="checked"</c:if> >
+                </td>
+                <td>
+                    <input class="visibilityCheckbox giverCheckbox${fqForm.question.questionNumber}"
+                        type="checkbox" value="<%= FeedbackParticipantType.OWN_TEAM_MEMBERS %>" <c:if test="!fqForm.isEditable">disabled="disabled"</c:if>
+                        <c:if test="${fqForm.generalSettings.giverNameVisibleFor['OWN_TEAM_MEMBERS']}"> checked="checked"</c:if> >
```

Comment:
May be a good idea to `c:set` an EL attribute for these constants.
Especially if they're relying on a constant from `Const`.